src/wr_control_drive_arm/src/JointStatePublisher.cpp

```diff
@@ -78,43 +82,49 @@ auto main(int argc, char **argv) -> int {
                                            RoboclawChannel::A,
-                                           getEncoderConfigFromParams(encParams, "elbow"),
+                                           getEncoderConfigFromParams(armParams, "elbow"),
+                                           getMotorConfigFromParams(armParams, "elbow"),
                                            n);
     namedJointPositionMonitors.try_emplace("elbowRoll_joint",
 
                                            "aux1",
                                            RoboclawChannel::B,
-                                           getEncoderConfigFromParams(encParams, "forearmRoll"),
+                                           getEncoderConfigFromParams(armParams, "forearmRoll"),
+                                           getMotorConfigFromParams(armParams, "forearmRoll"),
                                            n);
     namedJointPositionMonitors.try_emplace("shoulder_joint",
 
                                            "aux0",
                                            RoboclawChannel::B,
-                                           getEncoderConfigFromParams(encParams, "shoulder"),
+                                           getEncoderConfigFromParams(armParams, "shoulder"),
+                                           getMotorConfigFromParams(armParams, "shoulder"),
                                            n);
     namedJointPositionMonitors.try_emplace("turntable_joint",
 
                                            "aux0",
                                            RoboclawChannel::A,
-                                           getEncoderConfigFromParams(encParams, "turntable"),
+                                           getEncoderConfigFromParams(armParams, "turntable"),
+                                           getMotorConfigFromParams(armParams, "turntable"),
                                            n);
     namedJointPositionMonitors.try_emplace("wristPitch_joint",
 
                                            "aux2",
                                            RoboclawChannel::A,
-                                           getEncoderConfigFromParams(encParams, "wristPitch"),
+                                           getEncoderConfigFromParams(armParams, "wristPitch"),
+                                           getMotorConfigFromParams(armParams, "wristPitch"),
                                            n);
     namedJointPositionMonitors.try_emplace("wristRoll_link",
                                            "aux2",
                                            RoboclawChannel::B,
-                                           getEncoderConfigFromParams(encParams, "wristRoll"),
+                                           getEncoderConfigFromParams(armParams, "wristRoll"),
+                                           getMotorConfigFromParams(armParams, "wristRoll"),
                                            n);
```

Comment:
SUGGESTION: This may be abstract-able as well.

Reply:
Won't do (for now lol)